client/asset/interface.go

```diff
@@ -147,6 +147,8 @@ type Wallet interface {
 	Refund(coinID, contract dex.Bytes) (dex.Bytes, error)
 	// Address returns an address for the exchange wallet.
 	Address() (string, error)
+	// ValidateAddress indicates if an address belongs to the wallet.
+	ValidateAddress(address string) (bool, error)
```

Comment:
An address can be valid without being "owned" by the wallet. Maybe `OwnsAddress` instead?

Reply:
Agreed, in a sec.